### DISCUSSION TITLE
[IntelliSense] How can I get this intellisense outside markup

### DISCUSSION THREAD

**Discussion by benlieb** [tailwindlabs/tailwindcss#14097] [Ideas]
We have most of our tailwind classes inside a custom object like the following. Would it be possible to somehow get intellisense for this? 

<img width="640" alt="image" src="https://github.com/user-attachments/assets/863f3bbe-2e25-4db2-a645-0999e45fa512">

**Comment by wongjn** [comment]
You could try configuring the [`tailwindCSS.experimental.classRegex` setting](https://github.com/tailwindlabs/tailwindcss/issues/7553#issuecomment-735915659).

**Reply by benlieb** [reply]
Thanks for sharing this.

For posterity I got it to work with the following:

```javascript
  "tailwindCSS.experimental.classRegex": [
    ["getLcn\\(([^)]*)\\)", "'([^']*)'"],
    ["classnames\\(([^)]*)\\)", "'([^']*)'"]
  ]
```
